www/profileimage/index.php

```diff
@@ -0,0 +1,148 @@
+<?php
```

Comment:
Wait if this file is here, what is the big complicated block in the other profile file doing??